### ISSUE TITLE
Wrench doesn't work on any engines

### ISSUE THREAD

**Issue by PalatineKeepstar** [BuildCraft/BuildCraft#4525]
**Please complete the following information:**
 - Forge Version: 14.23.5.2847
 - BuildCraft Version (Add Hash if from github): 7.99.24.1
 - Any additonal mods: none

**To Reproduce**
1. place any engine
2. use wrench to change its direction
3. it doesn't work

**Comment by AEnterprise** [comment]
please provide a screenshot, there probably is nothing for the engine to output to

the wrench only makes it change between valid orientations, if there are no other valid orientations (meaning it will point to something that can accept MJ) it won't rotate at all

**Comment by xJon** [comment]
@AEnterprise That's true, although it's quite confusing in-game as there's still a sound effect playing even when it doesn't rotate.

**Comment by AlexIIL** [comment]
There's actually 2 sound effects that can play when you try to rotate an engine: 1 for when it was rotated, and 1 for when it couldn't be. However I don't think the sound effects are different enough to make it sound like it's not working (although it should be visually obvious when it doesn't work).

**Comment by xJon** [comment]
@AlexIIL Oh that's a good point, thanks for explaining. Yes I agree it's still a bit confusing, but not a real issue
